core/app/models/spree/shipping_method_zone.rb

```diff
@@ -0,0 +1,6 @@
+module Spree
+  class ShippingMethodZone < Spree::Base
+    belongs_to :zone
```

Comment:
Same question as other PR: Should we have an inverse of here, or is that detected successfully?